api/v1alpha1/drclusterconfig_types.go

```diff
@@ -27,12 +27,19 @@ type DRClusterConfigSpec struct {
 	// TODO: PeerClusters []ClusterID; to decide if we really need this!
 }
 
+const (
+	// DRClusterConfig configuration has been processed
+	DRClusterConfigConfigurationProcessed string = "Processed"
+	// DRClusterConfig S3 profile is reachable
+	DRClusterConfigS3Reachable string = "Reachable"
```

Comment:
We may need to change the condition name here from "Rechable" to something more descriptive. Can be part of the s3 PR that would follow.

For example as it stands the condition is reachable with reason reachable. The condition could be something like "MetadadataStoresReachable"